CppSQLite3.h

```diff
@@ -219,8 +222,11 @@ class CppSQLite3Table
     int getIntField(int nField, int nNullValue=0) const;
     int getIntField(const char* szField, int nNullValue=0) const;
 
-    double getFloatField(int nField, double fNullValue=0.0) const;
-    double getFloatField(const char* szField, double fNullValue=0.0) const;
+    float getFloatField(int nField, float fNullValue=0.0) const;
+    float getFloatField(const char* szField, float fNullValue=0.0) const;
+
+    double getDoubleField(int nField, double dNullValue=0.0f) const;
+    double getDoubleField(const char* szField, double dNullValue=0.0f) const;

```

Comment:
The default value for `fNullValue` should be `0.0f` with the `f` suffix while the default for `dNullValue` should be `0.0` without the `f` suffix. It looks like you have them swapped.

Reply:
Yeah just mixed them around.  I did add that but clearly just mixed where I added them to